Assets/Scripts/FakePlayerScript.cs

```diff
@@ -109,11 +102,28 @@ public class FakePlayerScript : MonoBehaviour {
 			} else {
 				
 				readyText.GetComponent<CanvasRenderer> ().SetAlpha (0.0f);
-				readyBG.GetComponent<CanvasRenderer> ().SetAlpha (0.0f);
-
-			}
+				readyBG.GetComponent<CanvasRenderer> ().SetAlpha (0.0f);
+
+            }
 
-		} else {
+            switch (playerIdentifier)
+            {
+
+                case 1:
+                    ChoosePlayerScript.Instance.player1Ready = true;

```

Comment:
Is this where you wanted to use the playerIdentifier to form the name of the property being changed?

Reply:
Yes! With JS I would just do `ChoosePlayerScript.Instance[`player${slot}Ready`] = true` and not need all these dang switches.